scripts/sct_create_mask.py

```diff
@@ -299,13 +299,14 @@ def create_line(param, fname, coord, nz):
     # set all voxels to zero
     sct.run(['sct_maths', '-i', 'line.nii', '-mul', '0', '-o', 'line.nii'], param.verbose)
 
-    cmd = ['sct_label_utils', '-i', 'line.nii', '-o', 'line.nii', '-create-add']
+    create_add_str = ""
     for iz in range(nz):
         if iz == nz - 1:
-            cmd += [str(int(coord[0])) + ',' + str(int(coord[1])) + ',' + str(iz) + ',1']
+            create_add_str += str(int(coord[0])) + ',' + str(int(coord[1])) + ',' + str(iz) + ',1'
         else:
-            cmd += [str(int(coord[0])) + ',' + str(int(coord[1])) + ',' + str(iz) + ',1:']
+            create_add_str += str(int(coord[0])) + ',' + str(int(coord[1])) + ',' + str(iz) + ',1:'
 
+    cmd = ['sct_label_utils', '-i', 'line.nii', '-o', 'line.nii', '-create-add', create_add_str]
```

Comment:
Is this a performance optimization? Or does it have to do with `argparse` interpreting the list version as too many arguments?

Reply:
It's closer to the latter, but not quite.

With the previous way, each coordinate would be added as a separate list element, so there ended up being an erroneous space between each coordinate. (i.e. You'd get `x1,y1,z1,val1: x2,y2,z2,val2` instead of the expected `x1,y1,z1,val1:x2,y2,z2,val2`.) This caused some parsing issues IIRC.